tools/psu/bd9e302efj.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+# Copyright 2021 The Racklet Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Tool for computing component constants for the ROHM Semiconductor BD9E302EFJ.
+# Component naming and computed values apply to the Application Example 2 (Fast load response)
+# in https://fscdn.rohm.com/en/products/databook/datasheet/ic/power/switching_regulator/bd9e302efj-e.pdf.
+
+import itertools
+import iec60063
+import math
+from tabulate import tabulate
+
+
+def avg(*args):
+    return sum(args) / len(args)
+
+
+# BD9E constants
+F_osc = 550000  # Hertz (this can't be changed)
+
+# Variables
+# TODO: Take these as command line arguments?
+V_in = 24  # Volts, input voltage
+A_max = 3  # Amperes, maximum output current
+dI_L = 1.0  # Amperes, inductor ripple current
+V_fb = 0.8  # Volts, feedback reference voltage
+G_mp = 20  # A/V, current sense gain
+G_ma = 140E-6  # A/V, error amplifier transconductance
+
+# Output capacitor C_out specifications
+C_C_out = 47E-6  # Farads, capacitance of C_out
+R_C_out = 2E-3  # Ohms, equivalent series resistance of C_out
+
+V_out_min = 5.1  # Volts, minimum voltage for V_out
+
+R_div_max = 700E3  # 700 KOhm maximum from the specification
+R_div_min = 500E3  # 500 KOhm minimum
+
+
+def compute_V_out(dR1, dR2):
+    return (dR1 + dR2) / dR2 * V_fb
+
+
+def generate_resistors(multipliers):
+    result = []
+    for m in multipliers:
+        result += [m * float(v) for v in iec60063.E96]
+    return result
+
+
+def resolve_V_out():
+    resolved = False
+    (b_voltage, b_R1, b_R2) = (math.inf, 0, 0)
+    for d_R1, d_R2 in itertools.permutations(generate_resistors([10000, 100000]), 2):
```

Comment:
Writing multipliers as multiples of thousands would improve readability
```suggestion
    for d_R1, d_R2 in itertools.permutations(generate_resistors([10E3, 100E3]), 2):
```